### ISSUE TITLE
Performance demo fails with error

### ISSUE THREAD

**Issue by nrempel** [hyperledger/aries-cloudagent-python#436]
Running the performance demo fails with the following error repeatedly:

```
Faber      | 2020-03-26 20:49:52,702 aries_cloudagent.core.conductor ERROR Exception in message handler:
Faber      | Traceback (most recent call last):
Faber      |   File "/home/indy/aries_cloudagent/core/dispatcher.py", line 169, in handle_message
Faber      |     await handler(context, responder)
Faber      |   File "/home/indy/aries_cloudagent/protocols/issue_credential/v1_0/handlers/credential_request_handler.py", line 54, in handle
Faber      |     cred_exchange_rec.credential_proposal_dict
Faber      |   File "/home/indy/aries_cloudagent/protocols/issue_credential/v1_0/manager.py", line 493, in issue_credential
Faber      |     if credential_definition["value"]["revocation"]:
Faber      | KeyError: 'revocation'
Faber      | 2020-03-26 20:49:52,710 aries_cloudagent.core.dispatcher ERROR Handler error: Dispatcher.handle_message
Faber      | Traceback (most recent call last):
Faber      |   File "/home/indy/aries_cloudagent/core/dispatcher.py", line 169, in handle_message
Faber      |     await handler(context, responder)
Faber      |   File "/home/indy/aries_cloudagent/protocols/issue_credential/v1_0/handlers/credential_request_handler.py", line 54, in handle
Faber      |     cred_exchange_rec.credential_proposal_dict
Faber      |   File "/home/indy/aries_cloudagent/protocols/issue_credential/v1_0/manager.py", line 493, in issue_credential
Faber      |     if credential_definition["value"]["revocation"]:
Faber      | KeyError: 'revocation'
Faber      | 2020-03-26 20:49:52,723 aries_cloudagent.core.conductor ERROR Exception in message handler:
Faber      | Traceback (most recent call last):
Faber      |   File "/home/indy/aries_cloudagent/core/dispatcher.py", line 169, in handle_message
Faber      |     await handler(context, responder)
Faber      |   File "/home/indy/aries_cloudagent/protocols/issue_credential/v1_0/handlers/credential_request_handler.py", line 54, in handle
Faber      |     cred_exchange_rec.credential_proposal_dict
Faber      |   File "/home/indy/aries_cloudagent/protocols/issue_credential/v1_0/manager.py", line 493, in issue_credential
Faber      |     if credential_definition["value"]["revocation"]:
Faber      | KeyError: 'revocation'
```

**Comment by swcurran** [comment]
@andrewwhitehead please investigate and fix.

**Comment by swcurran** [comment]
See this discussion @sklump had on HL Rocketchat -  https://chat.hyperledger.org/channel/aries-cloudagent-python?msg=ooMJDZE4tqBtxiSQS

The fix is in the discussion.

**Comment by swcurran** [comment]
Resolved with #437
